functions/api/v1/instance.ts

```diff
@@ -1,36 +1,93 @@
+// https://docs.joinmastodon.org/entities/Instance/
+// https://docs.joinmastodon.org/methods/instance/
 import type { Env } from 'wildebeest/backend/src/types/env'
 import { cors } from 'wildebeest/backend/src/utils/cors'
+import * as error from 'wildebeest/backend/src/errors'
 import { DEFAULT_THUMBNAIL } from 'wildebeest/backend/src/config'
 import { getVersion } from 'wildebeest/config/versions'
+import { calculateInstanceStatistics } from 'wildebeest/backend/src/mastodon/instance'
+import { MastodonInstance, InstanceStatistics } from 'wildebeest/backend/src/types/instance'
+import { MastodonAccount } from 'wildebeest/backend/src/types/account'
+import { loadLocalMastodonAccount } from 'wildebeest/backend/src/mastodon/account'
+import { Database, getDatabase } from 'wildebeest/backend/src/database'
+import { getAdmins } from 'wildebeest/backend/src/utils/auth/getAdmins'
+import { emailSymbol } from 'wildebeest/backend/src/activitypub/actors'
 
 export const onRequest: PagesFunction<Env, any> = async ({ env, request }) => {
-	const domain = new URL(request.url).hostname
-	return handleRequest(domain, env)
+	const domain: string = new URL(request.url).hostname
+	const dbOverride: Database = await getDatabase(env)
+	return handleRequest(domain, env, dbOverride)
 }
 
-export async function handleRequest(domain: string, env: Env) {
+export async function handleRequest(domain: string, env: Env, dbOverride?: Database) {
 	const headers = {
 		...cors(),
 		'content-type': 'application/json; charset=utf-8',
 	}
 
-	const res: any = {}
-
-	res.thumbnail = DEFAULT_THUMBNAIL
-
-	// Registration is disabled because unsupported by Wildebeest. Users
-	// should go through the login flow and authenticate with Access.
-	// The documentation is incorrect and registrations is a boolean.
-	res.registrations = false
+	const res: MastodonInstance = {
+		uri: domain,
+		title: env?.INSTANCE_TITLE,
+		description: env?.INSTANCE_DESCR,
+		short_description: env?.INSTANCE_DESCR,
+		email: env?.ADMIN_EMAIL,
+		version: getVersion(domain),
+		languages: ['en'],
+		registrations: env?.INSTANCE_ACCEPTING_REGISTRATIONS ?? false,
+		approval_required: env?.INSTANCE_REGISTRATIONS_REQUIRE_APPROVAL ?? false,
```

Comment:
registrations / approval_required should be hardcoded to false for now

Reply:
Done!